src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -1,38 +1,78 @@
 import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
 
 import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
-
 import { focusElement } from '~/platform/utilities/ui';
 
+import {
+  fetchMilitaryInformation as fetchMilitaryInformationAction,
+  fetchHero as fetchHeroAction,
+  fetchPersonalInformation as fetchPersonalInformationAction,
+} from '@@profile/actions';
+
+import { fetchTotalDisabilityRating as fetchTotalDisabilityRatingAction } from 'applications/personalization/rated-disabilities/actions';
+
+import ProfileHeader from 'applications/personalization/profile/components/ProfileHeader';
```

Comment:
I just realized we should eventually change the name of this component now that it's used outside of the Profile.

Reply:
I could make the change to `NameTag`, how does that sound? 😄

Reply:
works for me!

Reply:
Alright. I will add that update into the ticket to update the header in the profile.

Reply:
Awesome. There certainly isn't a rush on that.